### PR TITLE
File Upload for for Managed Account

### PR DESCRIPTION
**Pull request by monove**
Ability to upload a file to verify identity of a Managed Account as per https://stripe.com/docs/connect/identity-verification

Sample usage would be:
```
$stripe->fileUploads()->create($file,'identity_document','acct_38Bs8sca78mF5ec8r');
```

### CODE REVIEW

File: src/Api/FileUploads.php [hunk 1]
```diff
@@ -39,9 +39,9 @@ public function baseUrl()
      * @param  string  $purpose
      * @return array
      */
-    public function create($file, $purpose)
+    public function create($file, $purpose, $account = null)
```
Comment (GrahamCampbell) [inline src/Api/FileUploads.php]: You forgot to update the phpdoc for this method. ;)

File: src/Api/FileUploads.php [hunk 2]
```diff
@@ -39,9 +39,9 @@ public function baseUrl()
...
     {
-        $file = new PostFile('file', fopen($file, 'r'));
+        $file = new PostFile('file', fopen($file, 'r'), null, $account?['Stripe-Account'=>$account]:[]);
```
Comment (GrahamCampbell) [inline src/Api/FileUploads.php]: I think this codebase follows PSR-2 please.

File: src/Api/FileUploads.php (hunk 1 above)
Comment (monove) [inline src/Api/FileUploads.php]: Sorry, have very little time to spare here at work.
Feel free to scrap it and rebuild as you see fit. I've applied my changes locally.

**Comment by brunogaspar** [comment]
I will most likely do this in a different way.

**Comment by brunogaspar** [comment]
This is added on the 38b9e9cedd8fcae01935b9954a1a234bddb52502

Will be working on this for the 2.0.

**Pull request closed without merging**